### ISSUE TITLE
Handling attributes that state the field is null

### ISSUE THREAD

**Issue by theozaurus** [pdvrieze/xmlutil#69]
I'm trying to serialize some XML from a system I do not control. Any field with a null value is shown as follows:

```
  <address-line-1>Lombard Street</address-line-1>
  <address-line-2 nil="true"/>
```

Is it possible to write a policy that will correctly handle the attribute? Do you have any pointers on how I might do this?

**Comment by pdvrieze** [comment]
This is now available in the dev branch.

**Comment by pdvrieze** [comment]
Just released in 0.84.0-RC1
